scania-digital-design-system/tegel
# fix(table): adjust pagination textbox for three digit issue

## **Describe pull-request**  
This PR is about increasing the width of the table pagination input field. 

## **Issue Linking:**  
[CDEP-3433](https://tegel.atlassian.net/browse/CDEP-3433?atlOrigin=eyJpIjoiMTkyMDJmY2M2ZWUxNDE3ZTk2MGQ5NTQ0M2MwYmIwOTEiLCJwIjoiaiJ9)
[Github issue 674](https://github.com/scania-digital-design-system/tegel/issues/674)

## **How to test**  
_Provide detailed steps for testing, including any necessary setup._
1. Go to the aws storybook link
2. navigate to the table with pagination
3. Add 9999 pages
4. now try to enter in a four digit value in the input field and make sure its visible.

## **Checklist before submission**
- [x] I have added unit tests for my changes ( I have update snapshot)

## **Suggested test steps**


[CDEP-3433]: https://tegel.atlassian.net/browse/CDEP-3433?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
